blazor/pdfviewer/how-to/unexpected-token.md

```diff
@@ -58,4 +58,8 @@ public IActionResult Load([FromBody] Dictionary<string, string> jsonObject)
 
 Download the [Pdfium.dll](https://www.syncfusion.com/downloads/support/directtrac/general/ze/Pdfium1515619754).
 
-N> Use the `Syncfusion.EJ2.PdfViewer.AspNet.Core.Linux` package in your application for Linux environment. Also, ensure whether the library dependencies of lib pdfium. so they are installed properly. If not, execute the following command to install the RUN apt-get update\&& apt-get install -y --allow-unauthenticated \ libc6-dev \ libgdiplus \ libx11-dev \ curl \ vim \ supervisor \ pro cps
\ No newline at end of file
+N> Use the `Syncfusion.EJ2.PdfViewer.AspNet.Core.Linux` package in your application for Linux environment. Also, ensure whether the library dependencies of lib pdfium. so they are installed properly. If not, execute the following command to install the RUN apt-get update\&& apt-get install -y --allow-unauthenticated \ libc6-dev \ libgdiplus \ libx11-dev \ curl \ vim \ supervisor \ pro cps
+
+N> Only in WebAssembly applications can we set the ReferencePath. We can't set the ReferencePath in the Server application because of service dependencies that are not available. Only WebAssembly applications support service dependencies.
```

Comment:
N> Only in WebAssembly applications can you set the ReferencePath. You can't set the ReferencePath in the Server application because of service dependencies that are not available. Only WebAssembly applications support service dependencies.